pivy-importer/src/test/resources/deps/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
+Name: WMI
+Version: 1.4.8
+Summary: Windows Management Instrumentation
+Home-page: http://timgolden.me.uk/python/wmi.html
+Author: Tim Golden
```

Comment:
It seems weird to expose someone's name in a test resource file. Can you change his name and other sensitive info to dummy info?

Reply:
I think it should be fine, since this is a file from that package, we didn't change anything, just copy the file and leave the file intact.